doc/test-machine.md

```diff
@@ -31,32 +31,42 @@ to these services in a shared environment like uat/staging.
 (ns my.app-test
   (:require
     [my.app :as app]
-    [jackdaw.test :as j.t]
+    [jackdaw.serdes :refer [string-serde edn-serde]]
+    [jackdaw.serdes.json :as json-serde]
```

Comment:
I'm not a fan of the ns alias name, it's easily confused with a function name (like the 2 above). Can you name it `serdes.json` or even `jsj`

Reply:
OK, makes sense, thanks!